Demos/Priority/Allocator/PriorityAllocator.py

```diff
@@ -53,10 +53,23 @@ def allocate_path(self, bid: "PriorityPathBid", environment: "Environment", asta
 
         if bid.flying:
             if a.t != tick:
+                print(f"not next tick{bid.agent} - {tick}")
                 return None, None, f"Cannot teleport to {a} at tick {tick}."
+            allocated_segments = bid.agent.allocated_segments
+            if len(allocated_segments) > 0 and len(allocated_segments[-1].coordinates) > 0 \
+                    and allocated_segments[-1].coordinates[-1].inter_temporal_equal(a) \
+                    and allocated_segments[-1].coordinates[-1].t == a.t:
```

Comment:
why check intertemporal equality and then t separately?

Reply:
True